lib/targets/hip/device.cpp

```diff
@@ -56,6 +56,26 @@ namespace quda
       return device_count;
     }
 
+    void get_visible_devices_string(char device_list_string[128])
+    {
+      char *device_order_env = getenv("HIP_VISIBLE_DEVICES");
```

Comment:
I was using `ROCR_VISIBLE_DEVICES` on tioga, because somehow `HIP_VISIBLE_DEVICES` didn't make a difference. But I'm not entirely sure what affects which layer.

Reply:
I have zero experience on this ... maybe we should detect both `ROCR_VISIBLE_DEVICES` and `HIP_VISIBLE_DEVICES`?